lib/sensors/temperature.hpp

```diff
@@ -11,32 +11,55 @@
 
 namespace hyped::sensors {
 // Values and register names from data sheet
-static constexpr std::uint8_t kTemperatureDefaultAddress = 0x38;
-static constexpr std::uint8_t kCtrl                      = 0x04;
-static constexpr std::uint8_t kDataTemperatureHigh       = 0x07;
-static constexpr std::uint8_t kDataTemperatureLow        = 0x06;
-static constexpr std::uint8_t kStatus                    = 0x05;
+constexpr std::uint8_t kTemperatureDefaultAddress = 0x38;
+// Adding this since a singluar temperature sensor may be on a different address
+constexpr std::uint8_t kTemperatureAlternativeAddress = 0x3F;
```

Comment:
what is this? will sensors only have one of these 2 addresses? or are these the most common ones?

Reply:
Only one of these two

Reply:
make that clear in a comment i think

also singular not sigluar

Reply:
going to make it an enum so you can only ever put these two addresses

Reply:
on second thought, making the addresses come from an enum actually seems like a much worse solution considering the current state of the repl where we pass in an address manually (might need to change this to ask whether a default or alternative address is going to be read from for the temperature and accelerometer sensor)